app/app/src/main/java/com/example/myapplication/MainActivity.java

```diff
@@ -45,8 +45,8 @@
     Connector Headset = new Connector();
 
     boolean eegActive = false;
-    boolean carIsConnected = false;
-    boolean headsetIsConnected = false;
+    boolean carIsConnected;
+    boolean headsetIsConnected;
```

Comment:
This is false by default because the headset is not connected by default.

Reply:
It is only for testing between me and Simon. Did not intend to merge it.